tools/cmaketobzl/cmaketobzl.go

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright 2019 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/kythe/llvmbzlgen/cmakelib/ast"
+	"github.com/kythe/llvmbzlgen/cmakelib/bindings"
+	"github.com/kythe/llvmbzlgen/writer"
+)
+
+type blockCounter struct {
+	begin string
+	end   string
+	count int
+}
+
+func NewCounter(begin string) *blockCounter {
```

Comment:
I recommend unexporting these top-level bits in package main—there's no difference semantically, but it's a signal to the reader.

Reply:
Done for some, left for others (which I'd like to move to a separate package soon) .